web/src/components/Icons.js

```diff
@@ -9,7 +9,9 @@ import {
   faEdit,
   faExclamationTriangle,
   faPlusCircle,
-  faSpinner
+  faSpinner,
+  faUserCog,
+  faUserPlus
```

Comment:
If these are new to the project, we need to update the `LICENSE.md` to cite them.

Reply:
Jen already did when she closed the design issue! ^_^